src/components/tick.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+export default class Tick extends React.Component {
+  render() {
+    const {angle, x, y, length} = this.props;
+
+    return (
+      <g>
+        <line
+          transform={`rotate(${angle}, ${x}, ${y})`}
+          stroke="black"
+          strokeWidth="1"
+          x1={x}
+          x2={x}
+          y1={y}
+          y2={y - (length || 6)}
```

Comment:
if you have a default length it should be in a default prop, not just hanging out like that